io/src/test/java/org/commonjava/maven/ext/io/PomIOTest.java

```diff
@@ -70,7 +105,6 @@ public void testRewritePOMs()
         Project p = new Project( targetFile, model );
         HashSet<Project> changed = new HashSet<>();
         changed.add( p );
-
         pomIO.rewritePOMs( changed );
 
         assertTrue( FileUtils.contentEqualsIgnoreEOL( pom, targetFile, StandardCharsets.UTF_8.toString() ) );
```

Comment:
Charset could be `Charset.forName( model.getModelEncoding() ) `;

Reply:
I've moved everything to using StandardCharsets.UTF_8 as the model either has that or null (which breaks the tests)